content/en/blog/2021/revision-tags/index.md

```diff
@@ -6,19 +6,19 @@ attribution: "Christian Posta (Solo.io), Lin Sun (Solo.io), Sam Naser (Google)"
 
 To support this revision-based upgrade, Istio introduced the `istio.io/rev` label for namespaces which indicates to which control plane the workload data-plane proxies in that namespace should connect. For example, a label of `istio.io/rev=1-9-3` indicates the control plane revision `1-9-3` should inject the data plane for workloads in that namespace.
 
 If you wanted to upgrade the data-plane proxies for a particular namespace, you would update the `istio.io/rev` label to point to a new version such as `istio.io/rev=1-10-0`. Across many namespaces, updating labels one at a time could be tedious and error prone. Let's see how Istio can improve your experience doing upgrades.
 
 ## Introducing Revision Tags
 
-[In Istio 1.10](https://istio.io/latest/news/releases/1.10.x/announcing-1.10/), we've improved the revision feature [with a new feature called _revision tags_](https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels-experimental). A revision `tag` reduces the number of changes an operator has to make to use revisions and safely upgrade an Istio control plane. With a `tag`, you can point to a revision and use the `tag` as the label for your namespaces. That way you don't have to change the labels on a namespace and minimize the number of manual steps and configuration changes upgrading. Manually changing (or even trying to orchestrate) changes of labels across a large number of namespaces can be error-prone and lead to unintended downtime.
+[In Istio 1.10](/news/releases/1.10.x/announcing-1.10/), we've improved the revision feature [with a new feature called _revision tags_](/docs/setup/upgrade/canary/#stable-revision-labels-experimental). A revision `tag` reduces the number of changes an operator has to make to use revisions and safely upgrade an Istio control plane. With a `tag`, you can point to a revision and use the `tag` as the label for your namespaces. That way you don't have to change the labels on a namespace and minimize the number of manual steps and configuration changes upgrading. Manually changing (or even trying to orchestrate) changes of labels across a large number of namespaces can be error-prone and lead to unintended downtime.
 
 For example, you can define a `tag` named `prod-stable` and point it to the `1-9-3` revision of a control plane. You can also define another `tag` named `prod-canary` and also point it to a new `1-10-0` revision. You may have a lot of important namespaces in your cluster and can label those namespaces with `istio.io/rev=prod-stable`. You may have other namespaces where you are willing to canary control plane changes and can label that `istio.io/rev=prod-canary`. In both of these scenarios, the tag will indirectly associate those namespaces with the `1-9-3` revision for `prod-stable` and `1-10-0` for `prod-canary` respectively.
```

Comment:
Sorry, again not sure why I can't comment directly on the "revision tags in action" section but when you do `istioctl x revision tag list` after labeling a namespace with the `prod-stable` tag, the output should show that namespace next to the tag under the `NAMESPACES` column but doesn't seem to?

Also, at the end of that section when saying it's safe to remove the old revision, maybe document how to do that? (`istioctl x uninstall --revision 1-9-5` or whatever)